app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -505,6 +505,7 @@ private void onMediaRefreshed(Media media) {
                 .subscribe(this::onDepictionsLoaded, Timber::e)
         );
         // compositeDisposable.add(disposable);
+
```

Comment:
Any particular reason for this change? If not, would you mind removing the new line?

Reply:
I just added it by accident. I have removed it. Thank you!